### PR TITLE
2022 02 contextual help panel navigation

### PR DESCRIPTION
**Pull request by aurel-l**
## Purpose
Add content to contextual help panel (partly TRM-27303)

## Approach
 - Implement panel as its own routing context with a local in-memory history
 - Use the SlidingPanel's title to make a fake navigation bar (back/forward, home, expand)
 - "Landing"/home page of the panel to be added in further PR (fill with articles from page content)
 - Shortcut bar only on landing page
 - Search bar everywhere
 - Search page reusing similar approach as the existing HelpQuickSearch

## Testing
Manual testing. Tests fails because of issue in the spring branch, should fix itself when https://github.com/ebi-uniprot/uniprot-website/pull/530#discussion_r791982555 is fixed

## Checklist
- [x] My PR is scoped properly, and “does one thing only”
- [x] I have reviewed my own code
- [ ] I have checked that linting checks pass and type safety is respected (see comment above)
- [x] I have checked that tests pass and coverage has at least improved, and if not explained the reasons why

### CODE REVIEW

File: src/help/components/contextual/Shortcuts.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+} from '../../../app/config/urls';
+
+import styles from './styles/shortcuts.module.scss';
+
+// All of these exit the panel, so put them in the global history context
+const Shortcuts = ({ globalHistory }: { globalHistory: History }) => (
+  <Router history={globalHistory}>
+    <div className={styles.container}>
+      <Link to={LocationToPath[Location.HelpResults]}>
+        <HelpIcon width="1.5em" />
```
Comment (xwatkins) [inline src/help/components/contextual/Shortcuts.tsx:18]: width could be in a variable

File: src/help/components/contextual/NavigationBar.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import styles from './styles/navigation-bar.module.scss';
+
+type Props = {
+  localHistory: MemoryHistory;
+  globalHistory: History;
+};
+
+const NavigationBar = ({ localHistory, globalHistory }: Props) => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [_, setLocalLocation] = useState(localHistory.location);
```
Comment (xwatkins) [inline src/help/components/contextual/NavigationBar.tsx:18]: Why do we need this?

Comment (xwatkins) [reply src/help/components/contextual/NavigationBar.tsx:18]: (please add comment)

File: src/help/components/contextual/Shortcuts.tsx (hunk 1 above)
Comment (aurel-l) [reply src/help/components/contextual/Shortcuts.tsx]: Changed

File: src/help/components/contextual/NavigationBar.tsx (hunk 2 above)
Comment (aurel-l) [reply src/help/components/contextual/NavigationBar.tsx:21]: Added a lengthy comment explaining why this is needed

**Pull request merged**